docs/data/joy/getting-started/templates/email/components/EmailContent.tsx

```diff
@@ -37,20 +38,12 @@ export default function EmailContent() {
         }}
       >
         <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
-          <Box
-            component="img"
-            src="https://i.pravatar.cc/150?img=3"
-            sx={{
-              width: 40,
-              height: 40,
-              borderRadius: '8px',
-            }}
+          <Avatar
```

Comment:
The Avatar comes with smooth handling of the loading state compared to a `<img>` primitive.